src/QMCDrivers/MCPopulation.h

```diff
@@ -151,6 +151,7 @@ class MCPopulation
     auto walker_index = 0;
     for (int i = 0; i < walker_consumers.size(); ++i)
     {
+      walker_consumers[i]->initializeResources(trial_wf_->getResource());
```

Comment:
Crowds only need to clone and acquire the shared resources 1 time when they are constructed.  This is called after every population adjustment.  It confuses analysis to have important setup procedures occur the first time something is called.  Also I'm am not sure that trial_wf_ always has a valid resource because above I saw WalkerControl move it to a member unique pointer.

Reply:
I also expected creating resources once for the crowd. However, this is the only place I found access to both crowd and population. It ends up here.

I think this template distributeWalkers is a bit strange to me as well. Crowd and MCPopulation should know each other since a fat walker in a Crowd and MCPopulation should be identical.